services/generic/color_control.md

```diff
@@ -1,45 +1,104 @@
+> `sup_durations` property can contain an array of ranges with a specific step.
 
-#### Notes
+## Definitions
 
-- temp - is color temperature in Mired (micro reciprocal degree). It is related to Kelvins as:
-  `temp_kelvins = 1,000,000 / temp_mireds`
-  Supported `temp` values: 1-65279 mired. Actual color temperature supported by end devices is 2700K-6500K.
+* `color_component` represents a single color component, see the table below for well-defined color components:
 
-- warm_w - is warm white light source intensity. Value range 0-255.
+| Color component | Z-Wave | Zigbee | Range   | Description                                                                                                                    |
+|-----------------|--------|--------|---------|--------------------------------------------------------------------------------------------------------------------------------|
+| `red`           | Yes    | Yes    | 0-255   | Defines intensity of red color.                                                                                                |
+| `green`         | Yes    | Yes    | 0-255   | Defines intensity of green color.                                                                                              |
+| `blue`          | Yes    | Yes    | 0-255   | Defines intensity of blue color.                                                                                               |
+| `warm_w`        | Yes    | No     | 0-255   | Is intensity of warm white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `cold_w`        | Yes    | No     | 0-255   | Is intensity of cold white light source. Mix of warm white and cold white intensity forms color temperature.                   |
+| `temp`          | No     | Yes    | 1-65279 | Is the color temperature in Mired (micro reciprocal degree). Actual color temperature supported by end devices is 2700K-6500K. |
+| `amber`         | Yes    | No     | 0-255   | Defines intensity of amber color.                                                                                              |
+| `cyan`          | Yes    | No     | 0-255   | Defines intensity of cyan color.                                                                                               |
+| `purple`        | Yes    | No     | 0-255   | Defines intensity of purple color.                                                                                             |
 
-- cold_w - is cold white light source intensity. Value range 0-255.
+* `transition` is an object representing a single color transition change with the following structure:
 
-- Mix of warm white intensity and cold white intensity forms color temperature.
+| Field      | Type   | Example | Description                                         |
```

Comment:
Again here Example columns is not much helpful, Supported values would do the job and make spec shorter. 

| Field           | Type     | Supported values | Description                                                          |
|---------------|--------|---------------------|-----------------------------------------------------|
| component  | string | 'color_component' values | Selected color for transition                   |
| transition     | string | `"up", "down"`                    | Direction of the transition                       |
| duration        | int    | `<1-127>`                          | Optional. Duration of the transition in seconds    |

Reply:
It would need some more work for global change. I may nhelp you with it if you like.

Reply:
Supported values should be either in "Description" column if there are few of them or are referenced just once or in a designated point in Definitions section if this is a long list or are referenced multiple times. The whole problem was that before Example column was used either as example value or enum listing.

Supported values may of a field might be defined either as protocol constant or service property or otherwise. Description column is more flexible for this. I have spent a lot of time working on this template and I thought this through.

BTW: Your supported values column for duration is incorrect. The supported values range is defined in sup_durations service property. The only thing I would improve is a internal link to sup_components and sup_durations properties in the description column.

Reply:
Added links to props.